src/components/Body/Trials/Trials.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import TrialRow from './TrialRow/TrialRow';
+import classes from './Trials.module.css';
+
+const Trials = (props) => {
+
+    const allowedTrialsNumber = 8;
```

Comment:
it should be outside the component and since you will be creating a service file, please move it to the service file
also please rename it to TRIELS_NUMBER

Reply:
done